lopper/lops/lop-cpu-oslist.dts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2024 Advanced Micro Devices, Inc.  All rights reserved.
+ *
+ * Author:
+ *     Dhriti Sree Rompicharla <DhritiSree.Rompicharla@amd.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+*/
+
+/dts-v1/;
+
+/ {
+        compatible = "system-device-tree-v1,lop";
+        lops {
+                compatible = "system-device-tree-v1,lop";
+                lop_0 {
+                      compatible = "system-device-tree-v1,lop,select-v1";
+                      select_1;
+                      select_2 = "/.*:device_type:cpu";
+                      lop_0_1 {
+                              compatible = "system-device-tree-v1,lop,code-v1";
+                              code = "
+                                     import yaml
+                                     yaml_dict = {}
+                                     for node in __selected__:
+                                        cpu_ip_name =  node.propval('xlnx,ip-name')[0]
+                                        yaml_dict[node.label] = {'ip_name' : '','supported_os' : []}
+                                        yaml_dict[node.label]['ip_name'] = cpu_ip_name
+                                        if 'cortex' in cpu_ip_name or 'microblaze' in cpu_ip_name:
```

Comment:
I think this will add freertos also as supported os for Microblaze-V which is not true. MB-V contains ip-name as "microblaze_riscv".  Can you please cross check

Reply:
Hi DP, Thanks for the review. Updated the logic and force pushed. 
Please check it once.